src/components/organizations/OrgRoutes.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import { SearchBar as OrgSearchBar } from "./SearchBar";
+import { ViewOrganization } from "./ViewOrganization";
+import { CreateOrganization } from "./CreateOrganization";
+import { EditOrganization } from "./EditOrganization";
+import { regexMatch } from "../../utils/Switch";
+import { Router } from "../nav/Router";
+
+export const OrgRoutes = (url, props, router:Router) => 
```

Comment:
here is an example where we are taking some router (which is just a functional `switch` block) as input, along with the current url and props, and then we return a new router, one which builds on the input router by adding 4 cases (2 plain text matching cases and 2 regex matching cases), in decreasing priority order (the earlier match wins)